frontend/pages/hosts/components/DeleteHostModal/DeleteHostModal.tsx

```diff
@@ -77,7 +77,11 @@ const DeleteHostModal = ({
               newTab
             />
           </li>
-          <li>iOS and iPadOS hosts will re-appear unless MDM is turned off.</li>
+          <li>
+            {/* TODO(android): iOS, iPadOS, and Android hosts will re-appear unless MDM is turned
```

Comment:
Rather than add a feature flag check for this small copy change (which would require some needless prop drilling), I'm thinking we can just leave the old copy for now.